shell/packages/sandstorm-permissions/permissions.js

```diff
@@ -15,6 +15,18 @@
 // limitations under the License.
 
 var Crypto = Npm.require("crypto");
+var Url = Npm.require("url");
+
+SandstormPermissions = function (db) {
+  function cleanupSelfDestructing() {
+    var now = new Date();
+    db.collections.apiTokens.remove({expiresIfUnused: {$lt: now}});
+  };
+
+  // Make self-destructing tokens actually self-destruct, so they don't
+  // clutter the token list view.
+  SandstormDb.periodicCleanup(5 * 60 * 1000, cleanupSelfDestructing);
```

Comment:
I could just define `cleanupSelfDestructing()` here and move the registration to db-deprecated.js, but I'm not sure that's any better.